modules/reindex/src/main/java/org/elasticsearch/index/reindex/ReindexTaskStateDoc.java

```diff
@@ -87,6 +91,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.startObject();
         builder.field(REINDEX_REQUEST);
         reindexRequest.toXContent(builder, params, true);
+        if (allocationId != null) {
```

Comment:
I am a bit in doubt of what we need this for (think it ties in with the initial update)?